x-pack/legacy/plugins/spaces/public/views/management/edit_space/enabled_features/feature_table.tsx

```diff
@@ -105,6 +105,7 @@ export class FeatureTable extends Component<Props, {}> {
             aria-label={
               checked ? `${record.feature.name} visible` : `${record.feature.name} disabled`
             }
+            label=""
```

Comment:
**question** It seems like we're using `EuiSwitch` incorrectly now. It looks like we'll be [creating an empty label tag](https://github.com/elastic/eui/blob/79610c8ccd5fae172ede663bdb5ac5d69a0724d1/src/components/form/switch/switch.tsx#L81) next to the switch now.

If we're ok using switches without a label (like our use case here), should we instead be creating the markup without the `<label>` tag, and making the `label` prop optional?

![image](https://user-images.githubusercontent.com/3493255/63467546-b6289580-c433-11e9-9bce-8e55e6aceba0.png)

Reply:
Thanks, @legrego , I meant to call this out specifically.

Accessibility is the reason for the change (cc @myasonik), and I was going to run through the app and find the places with empty labels.

@dsnider, this seems like a design v. a11y situation, because now that I see it in use, the bare switch seems reasonable.

Reply:
@legrego We're going to roll back the changes to `EuiSwitch`. Not _just_ because of your concerns, but also some more sticky upgrade migration things we've run into too close to FF.

I'll be closing this PR in favor of one with an altered scope and version.

Reply:
Thanks for the heads-up, @thompsongl!